.github/workflows/linting.yml

```diff
@@ -14,9 +17,9 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
 
       - name: Set up Python 3.12
-        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5
```

Comment:
Were these files updated with `nf-core pipelines lint --fix`? Something does seem off about updating commit without versioned comment and downgrading Python version (I did not realize this was happing when you brought it up, sorry).

Could you either try to fix these files again, or create a new pipeline template with `nf-core pipelines create` and compare the Github actions files in the new pipeline repo to this one?

Reply:
Manually updated with a new create copy in 5033893e324b2ac8eb76c9684da3624fa8f5b8ef